tensorflow/core/kernels/BUILD

```diff
@@ -1608,7 +1657,9 @@ tf_kernel_libraries(
         "//tensorflow/core:nn_grad",
         "//tensorflow/core:nn_ops_op_lib",
         "//third_party/eigen3",
-    ],
+    ] + if_not_windows([
```

Comment:
Hmm..  As you can see the definition [here](https://github.com/meteorcloudy/tensorflow/blob/fe74dfaf000ecef0b36b847f50bb0283e8a8ac71/tensorflow/core/kernels/BUILD#L1602)
Looks like `math_now_windows` lacks dependencies `depthwise_conv_op` needs.
        `:conv_ops`,        `:ops_util`, `//tensorflow/core:nn_ops_op_lib`
Do you think it's fine to add them into deps of `math_not_window`?